packages/urls/src/urls.ts

```diff
@@ -117,3 +117,5 @@ export const CROWDIN_URL = 'https://crowdin.com/project/trezor-suite';
 
 export const TREZOR_SAFE_5_URL = 'https://trezor.io/trezor-safe-5';
 export const ESHOP_KEEP_METAL_SINGLE_SHARE_URL = 'https://trezor.io/trezor-keep-metal-single-share';
+
+export const COINMARKET_DOWNLOAD_INVITY_APP_URL = 'https://invity.onelink.me/yIY4/q7ltbnv0';
```

Comment:
If I would see this on hover in statusbar I would expect a phising website.

cc @marek-andrsk

Reply:
it allows us to see if users are actually downloading the app via Suite, which will be important for further iterations/improvements (or removal) of this page

Reply:
Can we use a more reliable domain, such as trezor.io/sth? .onelink.me/xyz looks like a phishing site.
cc @Hermez-cz

Reply:
Yes please, we need to avoid both looking phishy and/or teaching users it's OK.

Reply:
Also, will there be an analytics event, so we can statistically monitor the CTR?

Reply:
@Hermez-cz I don't think that's possible to embed into QR code, that's why we use Appsflyer link .. I found a way to use custom domain, processing this now, it will be via invity.io

Reply:
link finally changed